handler.go

```diff
@@ -2,21 +2,35 @@ package slogrollbar
 
 import (
 	"context"
+	"log/slog"
+	"net/http"
 	"time"
 
 	"github.com/rollbar/rollbar-go"
 	slogcommon "github.com/samber/slog-common"
-
-	"log/slog"
 )
 
+type wrappedError struct {
+	msg string
+	error
+}
+
+func (w wrappedError) Unwrap() error {
+	return w.error
+}
+
+func (w wrappedError) Error() string {
+	return w.msg
+}
+
 type Option struct {
 	// log level (default: debug)
 	Level slog.Leveler
 
 	// Rollbar client
-	Client  *rollbar.Client
-	Timeout time.Duration // default: 10s
+	Client     *rollbar.Client
+	Timeout    time.Duration // default: 10s
+	SkipFrames *int
```

Comment:
add it to the readme please

also: `// default: 2`

Reply:
Done!